Realm/RLMSyncManager.mm

```diff
@@ -164,20 +171,25 @@ + (void)_resetStateForTesting {
         }
     }
     [[s_sharedManager.activeUsers allValues] makeObjectsPerformSelector:@selector(logOut)];
+    NSURL *metadataURL = [s_sharedManager.fileManager fileURLForMetadata];
 
     // Reset the singleton.
+    s_testBundleID = nil;
     s_onceToken = 0;
```

Comment:
It's not sufficient to simply zero out a `dispatch_once_t` like this given how `dispatch_once` uses it.